stbt_auto_selftest.py

```diff
@@ -77,7 +80,6 @@ def main(argv):
     subparsers.add_parser('validate', help='Run (and check) the auto-selftests')
 
     cmdline_args = parser.parse_args(argv[1:])
-    screenshots_root = cmdline_args.screenshots_root or "selftest/screenshots"
```

Comment:
This isn't quite right.  I think that this should have been:

    if cmdline_args.screenshots_root is None:
        screenshots_root = os.path.join(root, "selftest/screenshots")
    else:
        screenshots_root = os.path.abspath(cmdline_args.screenshots_root)

As it is currently the `--screenshots-root` will always be taken relative to the test-pack root if it's a relative path.